content/admin/monitoring-managing-and-updating-your-instance/updating-the-virtual-machine-and-physical-resources/increasing-cpu-or-memory-resources.md

```diff
@@ -123,3 +123,15 @@ To increase the resources available to a {% data variables.product.prodname_ghe_
 {% data reusables.enterprise_installation.configuration-recognized %}
 {% data reusables.enterprise_installation.increase-resources-test-in-maintenance-mode %}
 {% data reusables.enterprise_installation.increase-resources-disable-message-and-maintenance-mode %}
+
+## Adding CPU or memory resources for Google Cloud Platform
+
+### Resizing considerations for Google Cloud Platform
+
+To add CPU or memory resources for an instance on Google Cloud, you must change the instance's size, have access to your company's Google Cloud infrastructure, and be familiar with  Google Cloud Run to manage Google Cloud instances.
+
+### Supported Google Cloud Platform instance sizes
+<!-- will work on tomorrow -->
+
+### Resizing for Google Cloud Platform
+<!-- will work on soon -->
```

Comment:
I'm also not sure what to put here @nguyenalex836.

Reply:
@mattpollard just a heads up on this comment from @CBID2! 💛